test/trento/domain/sap_system/sap_system_test.exs

```diff
@@ -537,6 +539,91 @@ defmodule Trento.SapSystemTest do
       )
     end
 
+    test "should not move an application instance if the host_id changed and the application instance is not clustered" do
```

Comment:
I would redo and rephrase this test. At the end, we want to check if already registered instance numbers in new not clustered hosts are registered as new instances:

`should register a new not clustered application instance if it has an already existing instance number`